packages/@lwc/integration-karma/scripts/karma-configs/hydration/base.js

```diff
@@ -36,14 +37,27 @@ const ALL_FRAMEWORK_FILES = [SYNTHETIC_SHADOW, LWC_ENGINE];
 process.setMaxListeners(1000);
 
 function getFiles() {
-    return [
+    const files = [
         ...(ENABLE_SYNTHETIC_SHADOW_IN_HYDRATION ? [createPattern(SYNTHETIC_SHADOW)] : []),
         createPattern(LWC_ENGINE),
         createPattern(TEST_SETUP),
         createPattern(TEST_UTILS),
         createPattern(TEST_HYDRATE),
-        createPattern('**/*.spec.js', { watched: false }),
     ];
+
+    // check if a .only file exists
+    const onlyFile = globSync('**/*/.only', { cwd: BASE_DIR, absolute: true });
+
+    if (onlyFile.length > 1) {
+        throw new Error(`More than one .only file found in ${BASE_DIR}`);
```

Comment:
Sometimes we want to test two things at once... I wouldn't throw here. I would just push multiple patterns if `onlyFile.length > 0`.

Reply:
Makes sense! I made the change and it seems to work well.